.gitignore

```diff
@@ -2,3 +2,4 @@
 /aws-nuke
 /aws-nuke-*
 .glide
+.idea
```

Comment:
We do not want to add any IDE specific ignores here. Every developer should use any IDE or text editor they want and requiring to update our `.gitignore` to reflect the use of all the quirky files some IDEs produce would be a cumbersome tasks.

You rather should [configure a global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

Reply:
Understood